include/class.mailfetch.php

```diff
@@ -675,6 +693,26 @@ function createTicket($mid) {
                     } catch (TnefException $ex) {
                         // Noop -- winmail.dat remains an attachment
                     }
+                    break;
+                // Parse attached email message
+                case 'message/rfc822':
+                    try {
+                        // Fetch the header of attached mime message.
+                        $body = $this->fetchBody($mid, $info['index'].'.0',
+                            $info['encoding']);
+                        // Add fake body to make the parser happy
+                        if ($body)
```

Comment:
Wonder is this was meant to be `if(!$body)`

Reply:
I already have this in my notes to discuss with you tomorrow 😂